recipes/certify/all/conanfile.py

```diff
@@ -38,7 +38,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.82.0")
+        self.requires("boost/[>=1.82 <2]")
```

Comment:
```suggestion
        self.requires("boost/1.83.0")
```

Boost is not allowed to use ranged versions, please, keep the latest version instead.

More info: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
Thanks!